### PR TITLE
[IMP] base,doc: re-add consolidation feature to gantt view

### PR DESCRIPTION
**Pull request by dmo-odoo**

### CODE REVIEW

File: doc/reference/views.rst [hunk 1]
```diff
@@ -1287,6 +1287,20 @@ take the following attributes:
     ``info``, ``muted``, ``primary``, ``success`` or ``warning``).
 ``default_group_by``
   name of a field to group tasks by
+``consolidation``
+  field name to display consolidation value in record cell
+``consolidation_max``
+  dictionary with the "group by" field as key and the maximum consolidation
```
Comment (aab-odoo) [inline doc/reference/views.rst]: dictionnary

File: doc/reference/views.rst [hunk 2]
```diff
@@ -1287,6 +1287,20 @@ take the following attributes:
...
+  value that can be reached before displaying the cell in red
+  (e.g. ``{"user_id": 100}``)
+
+  .. warning::
+      The dictionnary definition must use double-quotes, ``{'user_id': 100}`` is
+      not a valid value
+``consolidation_exclude``
+  name of the field that describe if the task has to be excluded
```
Comment (aab-odoo) [inline doc/reference/views.rst]: describes

File: doc/reference/views.rst [hunk 3]
```diff
@@ -1287,6 +1287,20 @@ take the following attributes:
...
+``consolidation``
+  field name to display consolidation value in record cell
+``consolidation_max``
+  dictionary with the "group by" field as key and the maximum consolidation
+  value that can be reached before displaying the cell in red
+  (e.g. ``{"user_id": 100}``)
+
+  .. warning::
+      The dictionnary definition must use double-quotes, ``{'user_id': 100}`` is
+      not a valid value
```
Comment (aab-odoo) [inline doc/reference/views.rst]: do we really need to add this constraint?

**Comment by dmo-odoo** [comment]
@aab-odoo I just put back what was there without actually checking it since we did keep all the same features. Thanks for the check. I think the warning is useful though, because the value is parsed in json and I personally faced the issue of mixing up the quotes and double quotes... ^^'

**Comment by aab-odoo** [comment]
robodoo r+

**Pull request closed without merging**